README.md

```diff
@@ -44,137 +44,93 @@ Once encrypted, the file looks like the following:
 
-### Vault.constructor
-
-The constructor of the `Vault` class.
+```bash
+$ kourou vault:encrypt config/prod/secrets.json --vault-key <password>
 
-```js
-Vault(vaultKey [, secretsFile, encryptedSecretsFile]);
+ 🚀 Kourou - Encrypts an entire file.
+ 
+ [✔] Secrets were successfully encrypted into the file config/prod/secrets.enc.json
 ```
 
-**Arguments**
+Then, you can securely store your secrets inside your repository and share them with you team. 
 
-| Name | Type              | Description |
-| -------- | ----------------- | ----------- |
-| `vaultKey`  | <pre>string</pre> | The key used to encrypt and decrypt secrets   |
-| `secretsFile`  | <pre>string</pre> | Optional secrets file   |
-| `encryptedSecretsFile`  | <pre>string</pre> | Optional encrypted secrets file   |
+To load the secrets inside an application, instantiate the Vault with the same password as for the encryption and the path to the secrets file.  
 
-**Properties**
-
-| Property | Type | Description |
-| `secrets` | <pre>string</pre> | The decrypted secrets after calling [decrypt](#decrypt) |
-
-#### Usage
+Then, use the decrypt method to load the secrets into the memory.  
 
 ```js
-const vault = new Vault('my vault key', 'secrets.json', 'secrets.enc.json');
-```
-
-___
-
-### Vault.decrypt
-
-Decrypt the content of the file designated by `encryptedSecretsFile` in the [constructor](#constructor) and store the decrypted content inside `secrets` of the `Vault` class.
-
-<br/>
+const vault = new Vault('password');
+vault.decrypt('config/prod/secrets.env.json');
 
-```js
-decrypt();
+// secrets are now available
+vault.secrets
 ```
 
-
-#### Usage
+You can also provide the password with the environment variable `KUZZLE_VAULT_KEY`.  
 
 ```js
-const vault = new Vault('my vault key', 'secrets.json', 'secrets.enc.json');
-vault.decrypt();
-console.log(vault.secrets); // Display decrypted secrets
-```
-
-___
+// process.env.KUZZLE_VAULT_KEY = 'password'
 
-### Vault.encrypt
+const vault = new Vault();
+vault.decrypt('config/prod/secrets.enc.json');
 
-Encrypt the content of the file designated by `secretsFile` in the [constructor](#constructor) and store the encrypted content in the file designated by `encryptedSecretsFile` in the [constructor](#constructor) or `outputFile` passed as argument. If the file exists it will be rewritten only if you set the argument `replaceFileIfExist` to `true`.
-
-<br/>
-
-```js
-encrypt([outputFile, replaceFileIfExist]);
+// secrets are now available
+vault.secrets
 ```
 
-**Arguments**
-
-| Name | Type              | Description |
-| -------- | ----------------- | ----------- |
-| `outputFile`  | <pre>string</pre> | Optional file used to store the encrypted secrets. If not set `encryptedSecretsFile` from the [constructor](#constructor) will be used instead |
-| `replaceFileIfExist`  | <pre>bool</pre> | Optional argument to overwrite the file if it already exists |
+### Strict mode
 
+In production, you may want to ensure that 
```

Comment:
incomplete documentation?

Reply:
I removed this part for now because I though it was too specific.  

This was the part about aborting if there is a key but not password or a password but no key

Reply:
Ok but... this sentence is cut short and is incomplete. As it is, this section doesn't make sense.

Reply:
(I forgot to push :expressionless: )